### PR TITLE
chore: bump apps engine version

### PR DESCRIPTION
**Pull request by AllanPazRibeiro**

### CODE REVIEW

File: .changeset/chatty-suits-dance.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+'@rocket.chat/license': minor
+'@rocket.chat/gazzodown': minor
+'@rocket.chat/ui-client': minor
+'@rocket.chat/livechat': minor
+'@rocket.chat/models': minor
+'@rocket.chat/cron': minor
+'@rocket.chat/meteor': minor
+---
+
+This updated version addresses issues with all applications utilizing the OAuth 2 library from Apps Engine, ensuring that apps like Google Drive and Google Calendar are operational once more.
```
Comment (sampaiodiego) [inline .changeset/chatty-suits-dance.md:34]:
```suggestion
Fix an issue affecting Rocket.Chat Apps utilizing the OAuth 2 library from Apps Engine, ensuring that apps like Google Drive and Google Calendar are operational once more.
```

**Review by sampaiodiego — CHANGES_REQUESTED** [review]

**Review by sampaiodiego — APPROVED** [review]

**Comment by swat-ordbogen** [comment]
Still issue with Google Calendar App

https://forums.rocket.chat/t/google-calendar-url-is-not-defined-at-oauth2client/19342

**Comment by sampaiodiego** [comment]
hi @swat-ordbogen .. just making sure, did you update to 6.6.6 and still had the issue with the google calendar app?

**Comment by swat-ordbogen** [comment]
> hi @swat-ordbogen .. just making sure, did you update to 6.6.6 and still had the issue with the google calendar app?

yes :-) 

Version
6.6.6
Apps Engine Version
1.41.1
Node Version
v14.21.3
Database Migration
304 (26. marts 2024 08:04)
MongoDB
6.0.6 / wiredTiger (oplog Enabled)

**Pull request merged**